cms/envs/common.py

```diff
@@ -2214,6 +2214,10 @@
 # in the blockstore-based XBlock runtime
 XBLOCK_RUNTIME_V2_EPHEMERAL_DATA_CACHE = 'default'
 
+# Blockstore data could contain S3 links, so this should be lower than Blockstore's
+# AWS_QUERYSTRING_EXPIRE (defaults to 3600)
+BLOCKSTORE_VALUE_CACHE_TIMEOUT = 169200
```

Comment:
I'm confused about the wording here; does `BLOCKSTORE_VALUE_CACHE_TIMEOUT` or `AWS_QUERYSTRING_EXPIRE` default to 3600?

Reply:
I think I'll just remove that part of the message then